data/scripts/merge_acs_data.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import csv
+import os
+import json
+
+
+if __name__ == '__main__':
+    # Parse input files from the command line
+    filenames = sys.argv[1:]
+    if len(filenames) == 0:
+        raise NameError(
+            'One or more arguments representing the files to transform are required'
+        )
+
+    fieldnames = ['fips', 'variable', 'fields']
```

Comment:
You can get a better sense of what each of these fields is supposed to represent in the `CensusObservation` model.